DinamicNavMenu/script.js

```diff
@@ -0,0 +1,15 @@
+const firstAnchor = document.getElementById("firstA");
+const secondAnchor = document.getElementById("secondA");
+const thirdAnchor = document.getElementById("thirdA");
+
+const title = document.getElementById("h1").innerHTML;
+
+if(title == "Home"){
+    firstAnchor.style.backgroundColor = "red";
+}
+if(title == "About"){
+    secondAnchor.style.backgroundColor = "green";
+}
+if(title == "Contact"){ 
+    thirdAnchor.style.backgroundColor = "pink";
+}
```

Comment:
o código não está formatado, era melhor usar os 3 iguais `===`  ,podes usar um switch para substituir esses if's
```suggestion
switch(tittle){
  case "Home":
    firstAnchor.style.backgroundColor = "red";
    break;
  case "About" :
    secondAnchor.style.backgroundColor = "green";
    break;
  case "Contact" :
    thirdAnchor.style.backgroundColor = "pink";
    break;
}
```